Makefile

```diff
@@ -52,8 +52,10 @@ GO_MODULE_PATH = $(shell go list -f '{{ .Dir }}' -m $(module))
 ##########################################################################
 # Configure Go build & test behaviors.
 
-# Enable the sqlite3 JSON extension.
-GO_BUILD_TAGS += json1
+# Tell the go-sqlite3 package to link against a pre-built library
+# rather than statically compiling in its own definition.
+# This will use the static library definitions provided by libbindings.a.
+GO_BUILD_TAGS += libsqlite3
```

Comment:
we don't need `json1`, because that affects the static library built by go-sqlite, and `libsqlite3` is asking it not to build that library in the first place.

This was motivated by linker errors. Updating the `rusqlite` crate (which provides the sqlite static library that we link in) cause new linker errors to come up from the Go compiler. This resolved them.